src/Socket.cpp

```diff
@@ -30,7 +30,7 @@
 #ifdef HAVE_LIBSSL
 // Copypasted from
 // https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28default.29
-// at 22 Dec 2014
+// at 2 Jan 2016
```

Comment:
There is ambiguity when you don't remember names and orders of months in English.